easypost/services/address_service.py

```diff
@@ -0,0 +1,73 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Union,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models.address import Address
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class AddressService(BaseService):
+    def __init__(self, client):
+        self.model_class = "address"
+
+    def create(
+        self,
+        verify: Optional[Union[Dict[str, Any], str, bool]] = None,
+        verify_strict: Optional[Union[Dict[str, Any], str, bool]] = None,
+        **params,
+    ) -> Address:
+        """Create an Address."""
+        url = self.class_url(self.model_class)
+
+        wrapped_params = {self.snakecase_name(self.model_class): params}  # type: Dict[str, Any]
```

Comment:
Suggestion: `self.snakecase_name` here is redundant because `model_class` is only a single word

Reply:
I'm going to apply the snakecase to everything. Check again, I updated it to use the actual class name which is uppercase and so this function will transform that.